### ISSUE TITLE
Search for mainainer

### ISSUE THREAD

**Issue by corvus-ch** [corvus-ch/rabbitmq-cli-consumer#81]
A little more than a year ago, I changed the company I am working with. This change also included a change in the technology I am working with. Professionally I do no longer have a need for `reabbitmq-cli-consumer` but I felt passionate about the project and I saw that people are using and were depending on it. So I tried to keep up the maintenance of it. Over time, it became more and more obvious, that I can no longer spare the time required to maintain this project. For that reason, I am looking for a person or a team willing to take over.

If you are interested in become a maintainer of this repository, please write ma an email and we can work out the details together.

**Comment by renanvieira** [comment]
I'm using this a lot in my current job and would love to help to keep the project alive.

**Comment by johnrhunt** [comment]
@corvus-ch Thanks for doing a great job, we use this tool a lot. It would be great if @renanvieira was made owner of it.

**Comment by johnduro** [comment]
@renanvieira have you any news from @corvus-ch ? I tried to reach him by mail but no chance.

We use a lot the consumer and it would be nice if you took it in your hand

**Comment by johnrhunt** [comment]
I was speaking to someone the other day who just uses long running PHP processes, he said he's had then running for > 18 months now with over 1000 messages/second.

One wonders if there's much substance to the 'PHP is meant to die' thing any more, what with the advent of PHP 7 etc.

It could be that a more wise move is to stop using this tool all toghether. That's not to say it's a bad tool (it's great), but is it still relevant? I genuinely don't know the answer.

**Comment by johnduro** [comment]
A new employee in our company told us the same thing but we also had issue on long running scrip not so long ago. 

It may depends on what/how you are doing or llibrary you are using but it may be worth a test because this tool is great but full (working) php would be better...

**Comment by rgerovski** [comment]
> I was speaking to someone the other day who just uses long running PHP processes, he said he's had then running for > 18 months now with over 1000 messages/second.

This is true only if we consider all messages process fast enough. Otherwise, 'PHP is meant to die' is still relevant in 2020: https://github.com/php-amqplib/RabbitMqBundle/issues/301

**Comment by renanvieira** [comment]
No, I didn't hear any news from @corvus-ch.

I read some cases on the internet about PHP7 being able to do long-running processes but I always had troubles, but to be honest, as far as I remember all applications that I had troubles with were ported from PHP5 to PHP7 without being rewritten.

But the fact that may _depends on what you are doing_ in the process is too risky to use in production. I prefer to use a stablished tool like this `rabbitmq-cli-consumer` than risky having known problems with PHP and long-running processes in production.

**Comment by corvus-ch** [comment]
@xmontero Yes, this is still the case.

@renanvieira It seems that I never received your emails and to my shame, I was not getting notifications from GitHub for some time, so I did not realize there was activity on this issue.

If somebody still wants to take over maintenance of this repository, please let me know by email. And as a cross-check if the email is not lost, please let me know here.

**Comment by corvus-ch** [comment]
This repository has been neglected long enough. I will now officially abandon this project by archiving it. As a preparatory work, all issues will be closed.
